### PR TITLE
CLI config command changes

### PR DESCRIPTION
**Pull request by Bowrna**
closes: https://github.com/apache/airflow/issues/22708
related: https://github.com/apache/airflow/issues/22708

Config command changes using click

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

**Review by blag — CHANGES_REQUESTED** [review]
This looks good, can you also convert/add the tests for this?

**Comment by Bowrna** [comment]
sure @blag I will add it and mark this PR ready for review.

File: airflow/cli/commands/config.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+def get_value(ctx, section, option ):
+    """Get one value from configuration"""
+    if not conf.has_section(section):
+        raise SystemExit(f'The section [{section}] is not found in config.')
+
+    if not conf.has_option(section, option):
+        raise SystemExit(f'The option [{section}/{option}] is not found in config.')
+
+    value = conf.get(section, option)
+    print(value)
```
Comment (blag) [inline airflow/cli/commands/config.py:68]: Let's use `rich.console.Console` to print things.

File: airflow/cli/commands/config.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+@click.pass_context
+@click_color
+def show_config(ctx, color):
+    """Show current application configuration"""
+    with io.StringIO() as output:
+        conf.write(output)
+        code = output.getvalue()
+        if should_use_colors(color):
+            code = pygments.highlight(code=code, formatter=get_terminal_formatter(), lexer=IniLexer())
+        print(code)
```
Comment (blag) [inline airflow/cli/commands/config.py:52]: Let's use rich.console.Console to print things.

**Review by blag — CHANGES_REQUESTED** [review]
Looks pretty good, although could use some fixups and tests.

File: airflow/cli/commands/config.py (hunk 1 above)
Comment (Bowrna) [reply airflow/cli/commands/config.py]: Screenshot 2022-06-17 at 9 06 47 PM
This is the print of the config without rich console
Screenshot 2022-06-17 at 9 07 11 PM
This is the print of the config with rich console. The section of the configuration is getting removed if I use rich console. I will see how to fix this issue. 
@blag

**Comment by Bowrna** [comment]
@blag I have made changes that you have suggested and this code is ready for review. Share your review when you get time. Thanks.

**Comment by blag** [comment]
Thank you for all of the work you've done for this.

From [this discussion](https://github.com/apache/airflow/pull/24591#pullrequestreview-1019268731), there is more groundwork and research necessary before we can evaluate rewriting the CLI with Click.

**Pull request closed without merging**